src/phone-number-input/data/country-code.ts

```diff
@@ -0,0 +1,262 @@
+// =============================================================================
+// Country model:
+// [
+//    Country name,
+//    Regions,
+//    iso2 code,
+//    International dial code,
+//    Format (if available),
+// ]
+// =============================================================================
+// Regions:
+// ['america', 'europe', 'asia', 'oceania', 'africa']
+// =============================================================================
+// Sub-regions:
+// ['north-america', 'south-america', 'central-america', 'carribean',
+//  'eu-union', 'ex-ussr', 'ex-yugos', 'baltic', 'middle-east', 'north-africa']
+// =============================================================================
+
+const countries = [
```

Comment:
We try to avoid default exports. So exporting it as `CountryData` might be favourable